### PR TITLE
Update Remove-ADGroupMember.md

### PR DESCRIPTION
**Pull request by HerbertMauerer**
As a result of windows bug "24359250: [RS1] Using Permissive Modify causes extra event 5136 to be written for Group membership changes" a new parameter was added to this cmdlet. It already appears in the online help in Powershell ISE.

### CODE REVIEW

**Comment by HerbertMauerer** [comment]
Hello,

I think this is stuck. Is there a ETA of this being released?
I have a customer waiting for it.

Herbert Mauerer
[cid:image001.png@01D6B99E.AA0872A0]<https://campus.partners.extranet.microsoft.com/sites/campus/Pages/wwcommunities.aspx>
Senior Escalation Engineer
Customer Services and Support EMEA
WW Communities Debugging<http://aka.ms/wwcomdebug>, SME
Which WW Community should you join? Click here<https://microsoft.sharepoint.com/teams/campusWWCs/Pages/Landing.aspx>
[cid:image002.jpg@01D6B99E.AA0872A0]
Do you have feedback for me? Please ask me for a connect survey.

**Comment by e0i** [comment]
Closing and re-opening to refresh validation checks.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by get-itips — CHANGES_REQUESTED** [review]
The file was updated after this PR was created so it is currently resulting in a merge conflict.
Please review the conflicts, thanks @HerbertMauerer !

**Comment by HerbertMauerer** [comment]
Please excuse me, this is one of my first and until now biggest changes I make in github directly. I made a few mistakes, and I have created new pull requests and the one where I seem to have gotten things right are 2228 and 2229.

So I think we should cancel this one. You can check the other PRs and edit review them. Thank you.

Herbert Mauerer

File: docset/windows/addsadministration/Remove-ADGroupMember.md [hunk 1]
```diff
@@ -307,6 +307,25 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisablePermissiveModify
+Group membership updates use permissive modify by default. This suppresses an error when removing a member that is not member of the group.
```
Comment (JohanFreelancer9) [inline docset/windows/addsadministration/Remove-ADGroupMember.md:311]:
```suggestion
Group membership updates use permissive modify by default. This suppresses an error when removing a member that is not a member of the group.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
If this PR is not closed, please include the one minor copy edit.

**Review by tseward — APPROVED** [review]

**Comment by e0i** [comment]
Closing based on the PR authors’ comments.

**Pull request closed without merging**